flink-connectors/flink-connector-elasticsearch5/pom.xml

```diff
@@ -82,6 +82,55 @@ under the License.
 			<version>${elasticsearch.version}</version>
 		</dependency>
 
+		<dependency>
+			<!-- Bump elasticsearch netty dependency -->
+			<groupId>io.netty</groupId>
+			<artifactId>netty-buffer</artifactId>
+			<version>4.1.44.Final</version>
```

Comment:
Smallest required jump.

Reply:
Hmm ok. I was just thinking if already do the effort to upgrade why not trying the latest version. But on the other hand, it might cause more problems.